Sources/FoundationEssentials/FileManager/FileManager+Utilities.swift

```diff
@@ -173,7 +173,12 @@ extension _FileManagerImpl {
         #endif
     }
 
-#if !os(Windows)
+#if os(Windows)
+#elseif os(WASI)
```

Comment:
Same question as above - would it be better to ensure we don't ever try to call this (like on Windows) rather than making it a no-op? That way we could throw a `.featureUnsupported` error if a developer tries to set it